Fix query urls for imdb.com and thetvdb.com
Based on tvheadend master branch
https://github.com/tvheadend/tvheadend/commit/d8a31e57a492be6628b685488fcc7f1d9d262679
https://github.com/tvheadend/tvheadend/commit/ce09077056f9c6558c188d135cec3be85cc9c200

--- a/src/webui/static/app/epg.js
+++ b/src/webui/static/app/epg.js
@@ -287,13 +287,11 @@ tvheadend.epgDetails = function(event) {
     win.show();
 
     function searchIMDB() {
-        window.open('http://akas.imdb.com/find?q=' +
-                    encodeURIComponent(event.title), '_blank');
+        window.open('https://www.imdb.com/find?q=' + encodeURIComponent(event.title), '_blank');
     }
 
     function searchTheTVDB(){
-        window.open('http://thetvdb.com/?string='+
-                    encodeURIComponent(event.title)+'&searchseriesid=&tab=listseries&function=Search','_blank');
+        window.open('https://www.thetvdb.com/search?query=' + encodeURIComponent(event.title), '_blank');
     }
 
     function playProgram() {

--- a/src/webui/static/app/dvr.js
+++ b/src/webui/static/app/dvr.js
@@ -87,13 +87,11 @@ tvheadend.dvrDetails = function(uuid) {
         }));
 
         function searchIMDB() {
-            window.open('http://akas.imdb.com/find?q=' +
-                        encodeURIComponent(title), '_blank');
+            window.open('https://www.imdb.com/find?q=' + encodeURIComponent(title), '_blank');
         }
 
         function searchTheTVDB(){
-            window.open('http://thetvdb.com/?string='+
-                        encodeURIComponent(title)+'&searchseriesid=&tab=listseries&function=Search','_blank');
+            window.open('https://www.thetvdb.com/search?query=' + encodeURIComponent(title),'_blank');
         }
 
         var win = new Ext.Window({
